/**:
    ros__parameters:
        log_root: "/home/lyx/git_ws/src/fusion_localization/result/"

        imu_topic: "/kitti/oxts/imu"
        gps_topic: "/kitti/oxts/gps/fix"

        acc_noise: 1e-2
        gyro_noise: 1e-4
        acc_bias_noise: 1e-6
        gyro_bias_noise: 1e-8

        # GPS到IMU外参
        I_p_Gps_x: 0.0
        I_p_Gps_y: 0.0
        I_p_Gps_z: 0.0












